tests/integration-tests/tests/update/test_update.py

```diff
@@ -1052,3 +1069,90 @@ def _test_shared_storages_mount_on_headnode(
         bucket_name,
         headnode_only=True,
     )
+
+
+def _test_shared_storage_rollback(
+    cluster,
+    existing_ebs_volume_id,
+    existing_ebs_mount_dir,
+    bucket_name,
+    new_ebs_mount_dir,
+    new_lustre_mount_dir,
+    new_efs_mount_dir,
+    remote_command_executor,
+    pcluster_config_reader,
+    scheduler_commands,
+    region,
+):
+    # update cluster with adding non-existing ebs and skip validator
+    problematic_volume_id = "vol-00000000000000000"
```

Comment:
We are covering the rollback on EBS mounting failure.
What about checking the rollback on other failures?

Reply:
Others are hard to check in integ test. Test with non-existing EFS, cluster update is blocked even if skip the validator. Tested with non-existing FSx, cluster update is blocked even if skip the validator. They are covered in manually test.

Reply:
Got it. Just for my better understanding: what do you mean with "cluster update is blocked even if skip the validator"? Shouldn't suppressing the validator be enough to submit an update that's going to fail?

Reply:
Suppress the validator doesn't work in this case